modules/ebi-metagenomics/mapseq/meta.yml

```diff
@@ -0,0 +1,56 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "mapseq"
+description: Perform taxonomic classification of rRNA reads using reference databases
+keywords:
+  - taxonomy
+  - reads
+  - otu
+  - rRNA
+tools:
+  - "mapseq":
+      description: "Open source metagenomic 16S/18S read classifier enabling comparative metagenomics."
+      documentation: https://github.com/jfmrod/MAPseq
+      doi: "10.1093/bioinformatics/btx517"
+      licence: "['dual-licensed under GPL v3+ or BSD 2-clause']"
```

Comment:
I can only see GPL 3 in the licence -> https://github.com/jfmrod/MAPseq/blob/master/COPYING

Reply:
huh interesting - this field was filled automatically from the bioconda repo: https://anaconda.org/bioconda/mapseq

The COPYING file in the github repo was last updated 6 years ago, maybe it's just outdated?

Reply:
ahh.. then I would just leave it. The code uses another tool which has a different licence, I couldn't find it... so I guess that is the reason for the mixed thingy.